pyanaconda/ui/gui/spokes/lib/accordion.py

```diff
@@ -551,4 +552,27 @@ def __init__(self, title, create_clicked_cb, autopart_type_changed_cb,
         combo.set_hexpand(False)
         combo.set_active_iter(default or store.get_iter_first())
         self._createBox.attach(combo, 0, 5, 2, 1)
+
+        label = Gtk.Label(
+            label=C_(
+                "GUI|Custom Partitioning|Autopart Page",
+                "_Automatically created mount points can be encrypted by default:"
```

Comment:
I know I should ask on the original PR but is this only me or the sentence doesn't feel right.

Reply:
@VladimirSlavik what do you think?

Reply:
I wanted to say "looks fine", but actually, no. I think it's one of these problems where you can't see it because you already know what it means.

I think dropping the "by default" part might help, but I'd rather bring in a native speaker...

Reply:
Quote from IRC:
```
<sbueno> jkonecny, the sentence itself is fine. what throws me off is the text next to the checkbox after
<sbueno> makes me do a doulbe take
<sbueno> so maybe saying something like: "Encrypt automatically created mount points by default:"
```